styles/chess.com/catppuccin.user.css

```diff
@@ -84,7 +84,7 @@
     @wq: #piece("w/wq") [];
     @wr: #piece("w/wr") [];
 
-    body {
```

Comment:
This is not what I said 😭 the "dark mode theme selector above" is on lines 23-25. *Remove* the selector and CSS block here and change the selector above to `.dark-mode body`. Because now this just won't work for light mode??

Reply:
i'll fix that